plugins/SitesManager/API.php

```diff
@@ -411,13 +411,18 @@ private function getSitesFromIds($idSites, $limit = false)
 
     protected function getNormalizedUrls($url)
```

Comment:
I know that code was already there but does it maybe belong rather in another class where it is also testable? Can be ignored for now if too much work